Resources/public/css/theme/views/fields/edit/image.css

```diff
@@ -37,3 +42,14 @@
 .ez-view-imageeditview .ez-image-properties-original {
     list-style-type: square;
 }
+
+.ez-view-imageeditview .ez-binarybase-help {
```

Comment:
it seems like it's the same for all *binary base view*, so instead of duplicating the style, this should go to `Resources/public/css/theme/views/fields/edit/binarybase.css`